src/platform/backends/lxd/lxd_request.cpp

```diff
@@ -56,6 +56,9 @@ const QJsonObject lxd_request_common(const std::string& method, QUrl& url, int t
         url.setQuery(project_query_string);
     }
 
+    if (url.scheme() == "http")
+        url.setScheme("https");
```

Comment:
It looks like an LXD test for an http url will be needed as well to achieve coverage for this, probably in tests/lxd/test_lxd_backend.cpp